.github/workflows/knative-go-test.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
     name: Unit Tests
     strategy:
       matrix:
-        go-version: [1.14.x]
+        go-version: [1.15.x]
```

Comment:
Will this pick the latest version always? I'm assuming the test bugs have been fixed in the latest release, but wanted to make sure.

Reply:
It should